src/components/topRail/multiplayerViewUsers/MultiplayerViewUsers.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-avatar-group class="container" v-if="!isMultiplayer">
+  <b-avatar-group class="container" v-show="!isMultiplayer">
```

Comment:
I think is better the v-if in this case. We don't want to render the template and this expensive component just to hide it

Reply:
This change was done @rodriquelca but maybe I can change it here too